docs/plans/2026-01-22-tywrapped-ecosystem-vision.md

```
@@ -0,0 +1,378 @@
+
+| Level | Indicator | Meaning |
+|-------|-----------|---------|
+| Basic | `latest` | Only current upstream version |
+| Standard | `latest+1` | Latest + previous major |
+| Full | `upstream` | Matches upstream's support matrix |
+| Extended | `upstream+LTS` | Upstream support + extended LTS |
+
+### Display Format
+
+README example:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)